std/datetime/date.d

```diff
@@ -146,7 +146,7 @@ public:
     /++
         Params:
             year   = The year portion of the date.
-            month  = The month portion of the date.
+            month  = The month portion of the date (January is 1).
             day    = The day portion of the date.
             hour   = The hour portion of the time;
             minute = The minute portion of the time;
```

Comment:
How about using the following format:
```
year   = The year portion of the date [-32,768 - 32,767].
month  = The month portion of the date [1 - 12].
day    = The day portion of the date [0 - 31].
hour   = The hour portion of the time [0 - 23].
minute = The minute portion of the time [0 - 59].
second = The second portion of the time [0 - 59].
```

Or alternatively as half-open interval, as in the documentation of [`std.datetime.date.TimeOfDay.this`](https://dlang.org/phobos-prerelease/std_datetime_date#.TimeOfDay.this)

```
year   = The year portion of the date [-32,768 - 32,768$(RPAREN).
month  = The month portion of the date [1 - 13$(RPAREN).
day    = The day portion of the date [0 - 32$(RPAREN).
hour   = The hour portion of the time [0 - 24$(RPAREN).
minute = The minute portion of the time [0 - 60$(RPAREN).
second = The second portion of the time [0 - 60$(RPAREN).
```

BTW, the [documentation](https://dlang.org/phobos-prerelease/std_datetime_date#.Date) states that the years range from 32,768 B.C. to 32,767 A.D., but since 1 B.C. is treated as 0, I think the docs should be corrected to **32,767** B.C. to 32,767 A.D. Probably @jmdavis can confirm.